gallery/visualtests/nxTransferListHalf.js

```diff
@@ -111,5 +116,21 @@ describe('NxTransferListHalf', function() {
     });
   });
 
+  describe('disable transfer', function() {
+    it('looks right', simpleTest(disableTransferListSelector));
+    it('does not put a dark border on hovered items',
+        hoverTest(disableTransferListSelector, secondDisableTransferItemSelector));
+    it('does not put a blue border and glow on focused items',
+        focusTest(disableTransferListSelector, secondDisableTransferItemSelector));
+    it('does not put a blue border and blue glow on focused+hovered items',
+        focusAndHoverTest(disableTransferListSelector, secondDisableTransferItemSelector));
```

Comment:
Given that the user can't actually focus the items I don't think these two tests are necessary